lib/StreamingStore.ts

```diff
@@ -27,16 +29,33 @@ implements RDF.Source<Q>, RDF.Sink<RDF.Stream<Q>, EventEmitter> {
   protected readonly store: S;
   protected readonly pendingStreams: PendingStreamsIndex<Q> = new PendingStreamsIndex();
   protected ended = false;
+  protected listeners: ListenerCallback[] = [];
+  public statusEventEmitter: EventEmitter | undefined;
 
   public constructor(store: RDF.Store<Q> = new Store<Q>()) {
-    this.store = <S> store;
+    this.store = <S>store;
```

Comment:
Change not needed?

Reply:
I just did `yarn lint --fix` and it produced those changes.